packages/core/test/utils-hoist/object.test.ts

```diff
@@ -444,7 +444,8 @@ describe('markFunctionWrapped', () => {
     const wrappedFunc = jest.fn();
     markFunctionWrapped(wrappedFunc, originalFunc);
 
-    expect((wrappedFunc as WrappedFunction).__sentry_original__).toBe(originalFunc);
+    // cannot wrap because it is frozen, but we do not error!
```

Comment:
huh, why did this test pass before? 😅 some ts bug or so? The new behavior is def. the actual expected behavior!